packages/lesmis/src/utils/useDropdownOptionsWithFavouriteDashboardItems.js

```diff
@@ -0,0 +1,32 @@
+import { useDashboardData } from '../api';
+import { useDashboardDropdownOptions } from './useDashboardDropdownOptions';
+import { useUrlParams } from './useUrlParams';
+
+const getDropdownOptionsWithFavouriteDashboardItems = (data = []) => {
+  const { favouriteDropdownOption, otherDropdownOptions } = useDashboardDropdownOptions();
+  const subDashboardsWithFavouriteDashboardItems = data.filter(
+    favouriteDropdownOption.componentProps.filterSubDashboards,
+  );
+
```

Comment:
This will map `dropdownOptions` with favourite dashboard items only, which helps us not to render unfavourite dashboard items.

We don't need to do that if we move `dropdownOptions` to backend, but I feel that will generate more work..

Reply:
Nah I think this is a good way to go 👍 seems a front-end concern